.github/workflows/continuous-integration.yml

```diff
@@ -100,6 +104,8 @@ jobs:
   tests:
     name: Tests
     runs-on: ubuntu-latest
+    env:
+      API_KEY: "not-a-real-api-key"
```

Comment:
This is required since the sdk will otherwise raise an exception, right ?

Reply:
correct